.github/workflows/integration-ci.yml

```diff
@@ -27,12 +27,12 @@ jobs:
 
       - name: start the database
         run: |
-          docker run --name my_postgres_db -e POSTGRES_PASSWORD=mysecretpassword -d -p 5431:5432 postgres
+          docker run --name my_postgres_db -e POSTGRES_PASSWORD=${{ secrets.POSTGRES_PASSWORD }} -d -p 5431:5432 postgres
```

Comment:
I don't think its worth putting `POSTGRES_PASSWORD` and `PSQL_CON_STR` inside environment secrets as they aren't sensitive information